src/fastlane/release_actions/lib/fastlane/plugin/release_actions/actions/calculate_next_release_version.rb

```diff
@@ -9,9 +9,10 @@ module Fastlane
   module Actions
     class CalculateNextReleaseVersionAction < Action
       def self.run(params)
-        tag = Git.last_release_tag
-        version = Version.from(tag)
-        messages = Git.log(tag)
+        release_tag_prefix = params[:release_tag_prefix]
+        last_tag = params[:from_tag].nil? ? Git.last_release_tag : params[:from_tag]
```

Comment:
What's the use case for overriding the last release tag?  

These changes look like they are non-breaking, so they should be okay, but we should be careful since this method is used by iOS as well.

Reply:
The implementation of `Git.last_release_tag` was not returning the correct tag when I was using the action in our repo (Something to do with how we tag our repo). Looks like there's been some updates since the last time I tried it, so let me give it another try. I'd rather not override stuff since it's shared, which is why I went with defaults that shouldn't break anything. 

Not having to change this would be ideal though, so let me give it a shot.